asterius/test/cloudflare.hs

```diff
@@ -5,12 +5,17 @@ import System.Process
 main :: IO ()
 main = do
   args <- getArgs
+  withCurrentDirectory "test/cloudflare" $ callCommand "npm install"
   callProcess "ahc-link" $
     [ "--bundle",
       "--browser",
       "--input-hs",
       "test/cloudflare/cloudflare.hs",
       "--input-mjs",
-      "test/cloudflare/cloudflare.mjs"
+      "test/cloudflare/cloudflare.mjs",
+      "--export-function=handleFetch",
+      "--ghc-option=-no-hs-main",
+      "--extra-root-symbol=Worker_x_closure"
```

Comment:
There doesn't seem to be a `Worker` module in this test, nor is there a top-level `x` closure?

Reply:
This flag was added ritualistically when trying to make `-no-hs-main` work, without understanding of what it actually does. I've removed the line and confirmed that tests pass.